### DISCUSSION TITLE
invalid game log: in kyoku 11: negative score

### DISCUSSION THREAD

**Discussion by chucklu** [Equim-chan/mjai-reviewer#136] [Q&A]
240414-5968d356-11bc-4d68-adb2-26da50908907_a217782092: invalid game log: in kyoku 11: negative score
invalid game log: in kyoku 11: negative score

四麻比赛，参照MLeague设置的规则，允许负分的。这边能支持分析有负分的对局吗？

**Comment by Equim-chan** [comment]
不计划支持这个。

**Reply by chucklu** [reply]
但是mleague是专业的比赛啊，无击飞的规则。参考官网https://m-league.jp/

**Comment by ryan070111** [comment]
想請問為什麼不支援負分？
是因為訓練資料已經有包含避免自己被擊飛的策略了嗎？
是否有其他辦法可以迴避這樣的策略呢？（例如當局每個人offset分數避免擊飛，或是分析前各補十萬點）

近期我這邊也有需要分析無擊飛規則牌譜的需求，期待你們的回覆，謝謝！

**Reply by Equim-chan** [reply]
最主要的原因是 feature encoding 没有考虑负分，负分将无法编码。
